### PR TITLE
[inductor] disable capture_pre_autograd_graph related UTs on Windows

### PR DESCRIPTION
**Pull request by xuhancn**
Contined to https://github.com/pytorch/pytorch/pull/132841

We disable `capture_pre_autograd_graph` related UT on Windows.
Disable `test_lstm_packed_change_input_sizes` and `test_multihead_attention` UTs on Windows.

**TODO:** 
Turn on them after fix `capture_pre_autograd_graph` issue on Windows.

## Local Test:
Linux is not skiped:
image

And we can skiped them on Windows:
image

cc @peterjc123 @mszhanyi @skyline75489 @nbcsm @iremyux @Blackhex @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @voznesenskym @penguinwu @EikanWang @Guobing-Chen @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

File: test/inductor/test_cpu_cpp_wrapper.py [hunk 1]
```diff
@@ -281,37 +292,37 @@ class BaseTest(NamedTuple):
             "test_qconv2d",
             "cpu",
             test_mkldnn_pattern_matcher.TestPatternMatcher(),
-            condition=torch.backends.mkldnn.is_available(),
+            condition=torch.backends.mkldnn.is_available() and run_not_on_Windows(),
```
Comment (jgong5) [inline test/inductor/test_cpu_cpp_wrapper.py:295]: simpler enough to just do `not IS_WINDOWS`?

Comment (xuhancn) [reply test/inductor/test_cpu_cpp_wrapper.py:298]: @jgong5 I use function is order to contains `TODO` comments for further turn on.

**Review by jgong5 — APPROVED** [review]
nit on the naming, other LGTM

File: test/inductor/test_cpu_cpp_wrapper.py [hunk 2]
```diff
@@ -42,6 +47,19 @@
 )
 
 
+def is_windows() -> bool:
```
Comment (desertfire) [inline test/inductor/test_cpu_cpp_wrapper.py:50]: Why not directly using `IS_WINDOWS` later?

Comment (xuhancn) [reply test/inductor/test_cpu_cpp_wrapper.py:50]: @desertfire I use function and mark `TODO` comments.

**Comment by xuhancn** [comment]
@desertfire use `IS_WINDOWS` now.

**Review by desertfire — APPROVED** [review]

**Pull request closed without merging**